projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -76,13 +80,28 @@ export class PaginationComponent {
    * @param type PaginationItemType
    * @returns string
    */
-  getAriaLabel(label?: string, type?: PaginationItemType): string {
+  getAriaLabel(item: PaginationItem): Observable<string> {
```

Comment:
I'm wary of changing this function so much because of clear breaking changes. But it might be ok if it was only recently introduced. Can you say when this was created? By us?

Reply:
I've added a feature flag since its an older method.